index.js

```diff
@@ -54,9 +37,26 @@ module.exports = function staticCache(dir, options, files) {
   }
 
   return function* staticCache(next) {
-    var file = files[safeDecodeURIComponent(path.normalize(this.path))]
-    if (!file)
-      return yield* next
+    var filename = safeDecodeURIComponent(path.normalize(this.path))
+    var file = files[filename]
+    if (!file) {
+      // hidden file
+      if (path.basename(filename)[0] === '.') return yield* next
+
+      var isExists = yield exists(path.join(dir, filename))
+      if (!isExists)
+        return yield* next
+
+      file = loadFile(filename, dir, options, files)
+    }
+
+    // when buffer to be false, load file every request
+    if (options.buffer === false) {
```

Comment:
here, when options.buffer to be `false`, will read file directly. And not cache header. This can use when local develop mode, and when options.buffer not defined, use old logic.